magento/magento2
# Fixed-20249: Added new pull request.Global select box arrow issue. wh…

### Description (*)

Backend: Global select box arrow issue. when user select any option wrong arrow icon displayed.
### Fixed Issues (if relevant)

1. magento/magento2 #20249>: Backend: Global select box arrow issue. when user select any option wrong arrow icon displayed.


### Manual testing scenarios (*)

1. Login into admin panel > Content > Configuration > Select any theme > Click on applied theme > choose any other theme. Arrow direction is wrong.

Note: This change for global select box.

2. Actual Result.
![actual](https://user-images.githubusercontent.com/43638601/51436436-4262b680-1cb3-11e9-9ea8-29a69f526758.png)

3. Expected Result.
![expected](https://user-images.githubusercontent.com/43638601/51436439-44c51080-1cb3-11e9-837a-c27891b6da6a.png)

### Contribution checklist (*)
